clang/lib/Headers/aiev2_core.h

```diff
@@ -1495,14 +1525,19 @@ inline v8caccfloat msc_elem_8(v8cfloat v1, v8cfloat v2,v8caccfloat acc) { return
 inline v8caccfloat mac_elem_8(v8float v1, v8cfloat v2, v8caccfloat acc)  { return mac_elem_8_accuracy_fast(v1,v2,acc);}
 inline v8caccfloat mac_elem_8(v8cfloat v1, v8float v2, v8caccfloat acc)  { return mac_elem_8_accuracy_fast(v1,v2,acc);}
 inline v8caccfloat mac_elem_8(v8cfloat v1, v8cfloat v2, v8caccfloat acc) { return mac_elem_8_accuracy_fast(v1,v2,acc);}
-inline v16accfloat mul_4x8_8x4(    v32float v1, v32float v2                                     ) { return mul_4x8_8x4_accuracy_fast(    v1, v2             );}
 inline v4caccfloat mul_2x8_8x2(    v16float v1, v16cfloat v2                                    ) { return mul_2x8_8x2_accuracy_fast(    v1, v2             );}
+#endif
+inline v16accfloat mul_4x8_8x4(v32float v1, v32float v2) {
+  return mul_4x8_8x4_accuracy_fast(v1, v2);
+}
 inline v16accfloat negmul_4x8_8x4( v32float v1, v32float v2                                     ) { return negmul_4x8_8x4_accuracy_fast( v1, v2             );}
 inline v16accfloat mac_4x8_8x4(    v32float v1, v32float v2, v16accfloat acc                    ) { return mac_4x8_8x4_accuracy_fast(    v1, v2, acc        );}
 inline v16accfloat msc_4x8_8x4(    v32float v1, v32float v2, v16accfloat acc                    ) { return msc_4x8_8x4_accuracy_fast(    v1, v2, acc        );}
 inline v16accfloat addmac_4x8_8x4( v32float v1, v32float v2, v16accfloat acc1, v16accfloat acc2 ) { return addmac_4x8_8x4_accuracy_fast( v1, v2, acc1, acc2 );}
-inline v16accfloat addmsc_4x8_8x4( v32float v1, v32float v2, v16accfloat acc1, v16accfloat acc2 ) { return addmsc_4x8_8x4_accuracy_fast( v1, v2, acc1, acc2 );}
-#endif
+inline v16accfloat addmsc_4x8_8x4(v32float v1, v32float v2, v16accfloat acc1,
+                                  v16accfloat acc2) {
+  return addmsc_4x8_8x4_accuracy_fast(v1, v2, acc1, acc2);
```

Comment:
Are there tests for these intrinsics?

Reply:
No there were no tests added for these emulation intrinsics.
One reason is that they only use already existing intrinsics, which are all covered by tests.
It would be better to have e2e execution tests for these intrinsics, but those should go somewhere else